dyno/panda/dyno_panel.py

```diff
@@ -0,0 +1,133 @@
+import tkinter as tk
+from tkinter import font
+from PIL import Image, ImageTk
+import serial
+import time
+
+#Define constants
+NUMSENSORS1 = 2 #RPM AND TEMPERATURE
+NUMSENSORS2 = 2 #JOULEMETER
```

Comment:
Nit: Is there any chance we can use more descriptive names here? Especially for constants that we want to be exposing to the user to configure the dyno setup. Also, is there a reason we aren't lumping both of these constants into something like a `num_runtime_inputs`? If they're handled differently, fair enough. Just wondering.

Reply:
The panda will need to loop through each serial port separately to read in the messages. So I believe we need to specify the constants separately. Will change the names though!